deliverylocation/models.py

```diff
@@ -0,0 +1,146 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from django.core.exceptions import FieldDoesNotExist
+from django.core.exceptions import ValidationError
+
+
+class User(models.Model):
```

Comment:
We have similar stuff on https://github.com/redhat-beyond/BuyTogether/pull/16/files. What's the reason to have it here as well?

Reply:
I know i have writted a comment at the begining,dont look at User SUpplier modules they are just helpers since the Supplier isnt merged yet,review other things and when all reviewd and fixed and supplier is merged i will rebase this PR on the upstream/main and update the paths and import the modules

Reply:
If we should "not look at something" it usually points to this PR being a Work In Progress (WIP). 
As I mentioned earlier, we'd like to refrain from reviewing PRs which will have major changes in the forseen future.

Reply:
@Omeramsc I didnt understand what do you mean,can you explain more?

Reply:
Sure: 
When you send a PR for review, you should respect the time of the reviewers (either mentors or random contributors on an open source project). usually when a PR is not complete and you just want someone to take a look and see you're going in the right direction, you'll mark it as a `work in progress`, a non-complete PR. 
If there's something we as reviewers should "not look at", it means this PR is indeed a work in progress.
If you'll mark it as one, I'll be able to review it in the right manner. otherwise I risk wasting time on reviewing something that will 99.9% change before this will be merged.

Reply:
> Sure: When you send a PR for review, you should respect the time of the reviewers (either mentors or random contributors on an open source project). usually when a PR is not complete and you just want someone to take a look and see you're going in the right direction, you'll mark it as a `work in progress`, a non-complete PR. If there's something we as reviewers should "not look at", it means this PR is indeed a work in progress. If you'll mark it as one, I'll be able to review it in the right manner. otherwise I risk wasting time on reviewing something that will 99.9% change before this will be merged.

should we use comment in the code to indicate WIP or some how in the git? in other words, how to highlight the code section as WIP?

Reply:
But i want reviewers to review the code and fix all and the only change i need to do is rebase this on the upstreammain untill the supplier module is merged so i can rebase and delete the module from the dilvery location,functions tests will be the same, the only thing that will changed is deletion of the extra code for the supplier module and i metion that in the comments

Reply:
@pazMenachem  It is not the part of code that is a WIP, but the whole PR. 
@thetaxas2468 So ask for reviewers anyway, but rebase is never "just rebase". things can change, and those things might have us review the PR again. 
so it is OK to ask for a review on a WIP or a PR that (heavily) depends on another, but you should mark it as so. It'll also make it easier to understand.

Reply:
How do i mark the whole PR as WIP and in the same time ask reviewers to review

Reply:
Edit the PR- there should be an option to mark it as a WIP
but even if there isn't, just add `WIP: ` at the beginning. 
then tag whoever you want to review as a reviewer, and explain yourself.

Reply:
Done